src/test/java/org/opensearch/knn/index/codec/KNN910Codec/KNN910CodecTests.java

```diff
@@ -5,17 +5,20 @@
 
 package org.opensearch.knn.index.codec.KNN910Codec;
 
+import org.junit.Ignore;
 import org.opensearch.knn.index.codec.KNNCodecTestCase;
 
 import java.io.IOException;
 import java.util.concurrent.ExecutionException;
 
 public class KNN910CodecTests extends KNNCodecTestCase {
 
+    @Ignore
```

Comment:
Should we delete the tests rather than marking them as ignore?

Reply:
Okay. If this is temporary thing, please create an issue and put the issue number here so that we don't miss it.